### PR TITLE
Add extraServices funcionality

### PR DESCRIPTION
**Pull request by marek-jiri**
For OKD is needed extra funcionality to create services

Signed-off-by: Jiri Marek <jiri.marek@dnation.cloud>

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -276,6 +276,16 @@ kube-prometheus-stack:
...
+extraServices: []
+  # - name: addservice
+  #   namespace:  targetnamespace
+  #   ports:
+  #   - name: portname
+  #     port: portnumber
+  #     targetPort: portnumber
+  #   selector:
+  #     app:app
+
```
[the remark this answers is not in the document]
Comment (marek-jiri) [reply chart/values.yaml:288]: add type

File: chart/templates/services.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+{{- end }}
+{{- if .externalTrafficPolicy }}
+  externalTrafficPolicy: {{ .externalTrafficPolicy }}
+{{- end }}
+  ports:
+{{ toYaml .ports | indent 2 }}
+{{- if .additionalPorts }}
+{{ toYaml .additionalPorts | indent 2 }}
+{{- end }}
+  publishNotReadyAddresses: {{ .publishNotReadyAddresses }}
```
[the remark this answers is not in the document]
Comment (marek-jiri) [reply chart/templates/services.yaml:41]: removed

File: chart/templates/services.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{{- range .Values.extraServices }}
```
[the remark this answers is not in the document]
Comment (marek-jiri) [reply chart/templates/services.yaml:1]: done

File: chart/templates/services.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+{{- end }}
+{{- if .loadBalancerIP }}
+  loadBalancerIP: {{ .loadBalancerIP }}
+{{- end }}
+{{- if .loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+  {{- range $cidr := .loadBalancerSourceRanges }}
+    - {{ $cidr }}
+  {{- end }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (marek-jiri) [reply chart/templates/services.yaml:32]: changed

**Review by chess-knight — APPROVED** [review]

**Pull request merged**